proxy/tools/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG BUILD_TAG=75238004b0fcfd8a7f71d380d7a774dda5c39622
+ARG ISTIO_TAG=1.23.0
```

Comment:
BUILD_TAG and ISTIO_TAG should not be hard-coded here because you will pass them in [proxy/tools/build.sh](https://github.com/opea-project/GenAIInfra/pull/363/files#diff-0b45998bbaa7553ce9065c91aad5a6c9f0c2e6d4163e9f9477ae3bc3b5767c2e)

Reply:
These tags only provide default values for the targeted release and will be overridden in build.sh.